src/components/Cell/Cell.tsx

```diff
@@ -1,14 +1,42 @@
 import React from "react";
 
-interface CellProps {
+export interface CellProps {
   row: number;
   column: number;
   isOpen: boolean;
+  isFlag: boolean;
+  isMine: boolean;
+  mineCount: number;
   onClick: (params: { row: number; column: number }) => void;
+  onClickContext: (params: { row: number; column: number }) => void;
```

Comment:
```suggestion
  onClick: () => void;
  onClickContext: () => void;
```

board에서 파라미터를 넘겨주어 ()=>void으로 해도 괜찮을듯 싶음